source/kubernetes/versions.rst

```diff
@@ -122,9 +122,9 @@ Upgrading Kubernetes Version
 ****************************
 
 When doing Kubernetes version upgrade, minor version cannot be skipped. For
-example, if the current cluster version is v1.21.x, then it's not allowed
-to upgrade directly to v1.23.x. You have to upgrade to v1.22.x and then do another
-upgrade to v1.23.x.
+example, if the current cluster version is v1.29.x, then it's not allowed
+to upgrade directly to v1.31.x. You have to upgrade to v1.30.x and then do another
+upgrade to v1.31.x.
```

Comment:
Probably worth linking to the Kubernetes skew policy here, and stating upgrades are in line with this.